autogluon_extra/src/autogluon/extra/task/__init__.py

```diff
@@ -4,8 +4,9 @@
 from autogluon.core.task.base import BaseTask
 from .image_classification import ImageClassification
 from .object_detection import ObjectDetection, Detector
-from .text_classification import TextClassification
-from .text_prediction import TextPrediction
+from autogluon.text.task.text_classification import TextClassification
```

Comment:
These can still be (and should be) relatively imported, such as `from ...text.task.text_classification import TextClassification`

Reply:
not if they are in different module

Reply:
Yes I believe they can be.